### ISSUE TITLE
RSS is missing autodiscovery

### ISSUE THREAD

**Issue by shishkin** [n0-computer/iroh.computer#255]
Hi! I was wondering why your blog has no RSS and was about to suggest adding it, but saw #153 and #219. Then I tried the RSS URL, but it returned no entries:

```bash
curl https://www.iroh.computer/rss
<?xml version="1.0" encoding="UTF-8"?><rss xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:content="http://purl.org/rss/1.0/modules/content/" xmlns:atom="http://www.w3.org/2005/Atom" version="2.0">
    <channel>
        <title><![CDATA[Blog posts | RSS Feed]]></title>
        <description><![CDATA[Welcome to this blog posts!]]></description>
        <link>https://iroh.computer</link>
        <image>
            <url>https://iroh.computer/img/logo/iroh-wordmark-purple.svg</url>
            <title>Blog posts | RSS Feed</title>
            <link>https://iroh.computer</link>
        </image>
        <generator>RSS for Node</generator>
        <lastBuildDate>Sun, 22 Dec 2024 13:49:11 GMT</lastBuildDate>
        <atom:link href="https://iroh.computer/rss" rel="self" type="application/rss+xml"/>
        <pubDate>Sun, 22 Dec 2024 13:49:11 GMT</pubDate>
        <copyright><![CDATA[All rights reserved 2024]]></copyright>
    </channel>
</rss>
```

Your site also lacks [RSS autodiscovery](https://www.rssboard.org/rss-autodiscovery), so no one will even know RSS is there.

**Comment by UnkwUsr** [comment]
For the time it is not working, here is workaround using service that makes rss feeds from web page: https://feedmaker.fly.dev/feed/?url=https%3A%2F%2Fwww.iroh.computer%2Fblog&feed_title=&selector_item=article&selector_title=.items-start+%3E+h2&selector_description=&selector_link=.items-start+%3E+h2+%3E+a

**Comment by dignifiedquire** [comment]
should be fixed now https://github.com/n0-computer/iroh.computer/pull/269

**Comment by UnkwUsr** [comment]
Confirmed, it works now on https://www.iroh.computer/rss . But what about RSS autodiscovery?

> Your site also lacks [RSS autodiscovery](https://www.rssboard.org/rss-autodiscovery), so no one will even know RSS is there.

https://github.com/n0-computer/iroh.computer/pull/258 tried to address this, probably including `<link rel="alternate" type="application/rss+xml" title="RSS" href="/rss" />` in head tag should be enough.

**Comment by dignifiedquire** [comment]
you are right, reopening until we have autodiscovery

**Comment by bltavares** [comment]
Indeed, https://github.com/n0-computer/iroh.computer/pull/258 should fix it, but it's closed. I can either open a new PR or someone else could try re-opening #258

**Comment by matheus23** [comment]
That PR has merge conflicts. If you want to have a stab at opening another PR to make this land sooner, that'd be amazing @bltavares

**Comment by shishkin** [comment]
I created #273
